source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=======================================
+Set Max Time MS for Aggregation Queries
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the :guilabel:`Max Time MS` option on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|. 
+
+About This Task 
+---------------
+
+By default, :guilabel:`Max Time MS` is set to 60000 milliseconds, or 60 seconds. Consider 
```

Comment:
[question, suggestion]

Here we talk about raising this value for large collections, but in the example we lower this value all the way down to 5 seconds. Maybe we should also mention a use case for lowering this value as well?

Reply:
Workshopped this and updated! Will check with external review to make sure this reasoning I added is accurate.